### PR TITLE
fix: Added missing type BranchMergeState

### PR DESCRIPTION
**Pull request by mascardoso**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -42,6 +42,19 @@ jobs:
...
+      - image: circleci/node:12-browsers
+    working_directory: ~/repo
+    steps:
+      - attach_workspace:
+          at: ~/repo
+      - restore_cache:
+          keys:
+            - v2-yarn-dependencies-{{ checksum "yarn.lock" }}
+            - v2-yarn-dependencies- # fallback to latest cache if no exact match is found
+      - run: DEBUG=abstract:* yarn check-ts-types
```
Comment (tommoor) [inline .circleci/config.yml:56]: You can't include the same key in the yaml file twice, the second one will overwrite the first. You want to add this `run` line within the existing `lint` job

**Comment by mascardoso** [comment]
Hi @tommoor @berezovskycom anything preventing from merging this, except for the conflicting files?

**Comment by williamw** [comment]
@tommoor I'm getting an error on the latest NPM version 8.0.0-beta.26.

 "error TS2304: Cannot find name 'BranchMergeState'." 

Will this fix that once you push to NPM?

**Comment by williamw** [comment]
Just updated to beta 28 and the error is gone.

**Pull request merged**